### PR TITLE
Remove NX and use react-native-bob instead

### PR DESCRIPTION
**Pull request by hcourthias**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,115 +1,125 @@
 {
-  "name": "streamline",
-  "version": "0.0.0",
-  "license": "MIT",
+  "name": "@getluko/streamline",
+  "version": "0.0.53",
```
Comment (judastheo) [inline package.json:3]: 0.1.0

File: package.json [hunk 2]
```diff
@@ -1,115 +1,125 @@
...
-    "prepare": "husky install",
-    "postinstall": "patch-package",
-    "sandbox:start": "yarn nx run sandbox:start",
-    "sandbox:start:e2e": "EXPO_PUBLIC_E2E=true yarn nx run sandbox:start-e2e",
-    "sandbox:start:clear": "yarn nx run sandbox:start --clear",
-    "sandbox:test": "yarn nx run streamline:test",
-    "sandbox:test:visual:ios": "yarn nx run sandbox:test-visual-ios",
-    "sandbox:test:visual:ios:compare": "yarn nx run sandbox:test-visual-ios-compare",
-    "sandbox:test:visual:android": "yarn nx run sandbox:test-visual-android",
-    "sandbox:test:visual:android:compare": "yarn nx run sandbox:test-visual-android-compare",
```
Comment (judastheo) [inline package.json:15]: What about all these scripts ?

File: package.json [hunk 3]
```diff
@@ -1,115 +1,125 @@
 {
-  "name": "streamline",
-  "version": "0.0.0",
-  "license": "MIT",
+  "name": "@getluko/streamline",
+  "version": "0.0.53",
   "scripts": {
-    "prepare": "husky install",
-    "postinstall": "patch-package",
```
Comment (judastheo) [inline package.json:7]: These two can stay as is

**Review by judastheo — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (hcourthias) [reply package.json:3]: I will do it when the github action for publishing to npm is ready

File: package.json (hunk 2 above)
Comment (hcourthias) [reply package.json:15]: you can do `yarn sandbox test:visual:ios` instead

File: package.json (hunk 3 above)
Comment (hcourthias) [reply package.json:7]: we do not use patch-package anymore (`yarn patch` instead).
For pre-commit, we use lefthook which came pre-configured with bob

**Pull request merged**